addons/binding/org.openhab.binding.snapcast/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="snapcast"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="serverStreams">
```

Comment:
I think that these channels would make more sense if they are provided as a selectable list of Options, see for example the Spotify binding:

https://github.com/openhab/openhab-addons/blob/2.5.x/bundles/org.openhab.binding.spotify/src/main/java/org/openhab/binding/spotify/internal/handler/SpotifyDynamicStateDescriptionProvider.java

Can you take a look at that? If that does not match what you need please explain your usecase.